crates/host/src/error.rs

```diff
@@ -0,0 +1,43 @@
+use holochain_wasmer_common::WasmError;
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+
+/// Wraps a WasmErrorInner with a file and line number.
+/// The easiest way to generate this is with the `wasm_error!` macro that will
+/// insert the correct file/line and can create strings by forwarding args to
+/// the `format!` macro.
+#[derive(Clone, Debug, PartialEq, Eq, Hash, PartialOrd, Ord, Serialize, Deserialize, Error)]
+#[rustfmt::skip]
+pub struct WasmHostError(pub WasmError);
```

Comment:
A wrapper error type around WasmError lets us put the type conversion to wasmer::RuntimeError in `holochain_wasmer_host`.

The only place that conversion is used is in the `holochain` crate ribsome, which already imports `holochain_wasmer_host`